core/src/main/scala/cats/syntax/functor.scala

```diff
@@ -1,4 +1,34 @@
 package cats
 package syntax
 
+import cats.evidence.<~<
+
 trait FunctorSyntax extends Functor.ToFunctorOps
+
+private[syntax] trait FunctorSyntaxBinCompat0 {
```

Comment:
Is there a reason you can't just add this to `FunctorSyntax` and skip all the `BinCompat` stuff? It shouldn't be necessary now that we don't support Scala 2.11 on master.

Reply:
@travisbrown I wasn't too sure so I left that as-is. Will move `catsSyntaxFunctorOps0` into `FunctorSyntax`.